src/figuresClasses.ts

```diff
@@ -1,19 +1,87 @@
-export interface Figure {
+type Shape = 'triangle'| 'circle'| 'rectangle';
+type Color = 'red'| 'green'| 'blue';
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
+
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    const longestSide = [a, b, c].sort((side1, side2) => side2 - side1);
```

Comment:
```suggestion
    const longestSide = [a, b, c].sort((side1, side2) => side2 - side1)[0];
```
OR
```suggestion
    const sortedSides = [a, b, c].sort((side1, side2) => side2 - side1);
```
to make name matches with value, just FYI

Reply:
Clear, thank you.